packages/sources/eth-balance/test/integration/balance.test.ts

```diff
@@ -86,4 +86,27 @@ describe('execute', () => {
       expect(response.body).toMatchSnapshot()
     })
   })
+
```

Comment:
Might also want to add tests here for when minConfirmations is out of bounds (<0 or >64)

Reply:
This is already covered in the unit tests 👍

Reply:
Oh I see this now, ty